silhouette/app/com/mohiva/play/silhouette/impl/providers/OpenIDProvider.scala

```diff
@@ -140,6 +141,7 @@ trait OpenIDService {
  *
  * @param providerURL The OpenID provider URL used if no openID was given. @see https://willnorris.com/2009/07/openid-directed-identity-identifier-select
  * @param callbackURL The callback URL to the application after a successful authentication on the OpenID provider.
+ *                    The URL can be a relative path which will be resolved using against a request's host.
```

Comment:
I'm not an English pro. But does this really means "be resolved using against" and not "be resolved against" without the "using"?

I think it sounds better:
The URL can be a relative path which will be resolved against the current request's host.

What do you think?